src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -68,13 +70,30 @@ def ingest(
     :param output_ext: extension for the output images in tiledb
     :param tile_scale: The scaling factor applied to each tile during I/O.
         Larger scale factors will result in less I/O operations.
-    :param access_credentials_name: Access Credentials Name (ACN) registered
-        in TileDB Cloud (ARN type)
+    :param access_credentials_name: [TBDeprecated] Access Credentials Name (ACN)
+        registered in TileDB Cloud (ARN type) if `acn` is not set.
```

Comment:
nit: two <code>``</code>s in ReST-style comments (since one <code>`</code> has its own meaning).